pkg/operator/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -231,7 +227,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	// for that new namespace. This allows us to be up and running for other components that don't
 	// yet exist, but will.
 
-	namespaceMapFn := handler.EnqueueRequestsFromMapFunc(func(a client.Object) []reconcile.Request {
+	namespaceMapFn := handler.EnqueueRequestsFromMapFunc(func(ctx context.Context, a client.Object) []reconcile.Request {
 		// Iterate all CredentailsRequests to determine if we have any that target
 		// this new namespace. We are not anticipating huge numbers of CredentialsRequests,
 		// nor namespace creations.
```

Comment:
Pass the context you got in the method signature into the List() below

Reply:
done in upcoming commit